Packs/Jira/Integrations/JiraV3/JiraV3.py

```diff
@@ -3810,18 +3838,44 @@ def get_issue_id_or_key(issue_id: str = '', issue_key: str = '') -> str:
     return issue_id or issue_key
 
 
+def validate_auth_params(
+    username: str, api_key: str, client_id: str, client_secret: str
+) -> None:
+    basic_oauth, oauth2 = username or api_key, client_id or client_secret
```

Comment:
the variable names`basic_outh` and `oauth2` are confusing, consider `credential_key`, `credential_value` or something similar